### PR TITLE
Added numpy average frontend function and tests

### PR DESCRIPTION
**Pull request by shireenchand**
Issue number - https://github.com/unifyai/ivy/issues/6739

### CODE REVIEW

**Comment by vaatsalya123** [comment]
Hey @shireenchand thanks for contributing to ivy.Your pr needs a few changes that I have requested you might need to fix these before we can merge this (:

File: ivy/functional/frontends/numpy/statistics/averages_and_variances.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import ivy
 from ivy.func_wrapper import from_zero_dim_arrays_to_float
 from ivy.functional.frontends.numpy.func_wrapper import to_ivy_arrays_and_back
+import pytest
```
Comment (vaatsalya123) [inline ivy/functional/frontends/numpy/statistics/averages_and_variances.py:5]: We don't import pytest into the frontends directly provided your not using pytest.mark_parametrize.We just rely on hypothesis testing.

**Comment by vaatsalya123** [comment]
You have also committed your local changes to the .idea folder please uncommit that from your changes.

**Comment by shireenchand** [comment]
Hi @vaatsalya123, I have made the changes requested. Please check and let me know if there are any other changes
Thanks!

**Comment by vaatsalya123** [comment]
@shireenchand you still have changes to the .idea folder which have not been uncommitted.Also like I explained we do not import pytest into the frontend for any implementation of a function.Try and avoid using pytest in your function implementation please thanks.

**Comment by shireenchand** [comment]
@vaatsalya123 can't pytest be used in the testing for frontend part?

**Comment by vaatsalya123** [comment]
provided that you are not using pytest.mark_parametrize, it makes no sense for you to use pytest in the frontends.We rely on hypothesis test for the frontends.If you look through other implementations in the frontends none use pytest and that's our standard format.

**Comment by shireenchand** [comment]
@vaatsalya123 I have made a new PR - https://github.com/unifyai/ivy/pull/7347
I have made all the changes requested. I had to delete this current PR due to the polluted got history.
Please do review it
Thanks!

**Pull request closed without merging**